scrapenews/spiders/timeslive.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+from .sitemap import SitemapSpider
+from scrapenews.items import ScrapenewsItem
+from datetime import datetime
+import pytz
+
+SAST = pytz.timezone('Africa/Johannesburg')
+
+
+class TimesliveSpider(SitemapSpider):
+    name = 'timeslive'
+    allowed_domains = ['www.timeslive.co.za']
+
+    sitemap_urls = ['https://www.timeslive.co.za/robots.txt']
+    sitemap_follow = [
+        'www.timeslive.co.za/anc-conference-2017',
+        'www.timeslive.co.za/news',
+        'www.timeslive.co.za/politics',
+        'www.timeslive.co.za/investigations',
+        'www.timeslive.co.za/opinion-and-analysis',
+        'www.timeslive.co.za/business',
+    ]
+
+    publication_name = 'Times Live'
+
+    def parse(self, response):
+        
+        title = response.xpath('//h1/span/text()').extract_first()
+        self.logger.info('%s %s', response.url, title) # check what this does
+        article_body = response.xpath('//div[@class="article-widget article-widget-text"]')
+        if article_body:
+            body_html = article_body.extract_first()
+            body_text = "\n\n".join(response.xpath('//div[@class="article-widget article-widget-text"]/div/div/p/text()').extract()) # currently unused but depending on ultimate use of body_html might this be better?
+            
+            byline_includes_by = response.xpath('//span[@class="heading-author"]/text()').extract()
+            byline_str = " ".join(byline_includes_by)
+            byline_no_by = (byline_str).split(" ")[1:]
+            byline = " ".join(byline_no_by)
+            
+            pubdate_long = response.xpath('//div[@class="article-pub-date "]/text()').extract()[0].strip()
+            day = pubdate_long.split(" ")[0]
+            month = pubdate_long.split(" ")[1]
+            if month == 'January':
```

Comment:
why are you recoding months instead of using `%B`? https://docs.python.org/2/library/datetime.html#strftime-strptime-behavior

Reply:
In fact you can probably put pubdate_long straight into strptime with all the spaces and the hyphen between the format parameters.

But don't stress about getting this fixed up - let's just get it working then we can improve the code :)